### ISSUE TITLE
Getting a file_locking error in the server log file when uploading a file - preview tile is missing

### ISSUE THREAD

**Issue by jnfrmarks** [owncloud-archive/files_locking#23]
### Steps to reproduce
1. Install 8.1
2. upload a file
### Expected behaviour

No file locking errors thrown
### Actual behaviour

A file locking error is thrown:

```
{"reqId":"WC6GIL9YU44WNUgBf82b","remoteAddr":"x.x.x.x","app":"index","message":"Exception: {\"Exception\":\"OCP\\\\Files\\\\LockNotAcquiredException\",\"Message\":\"Could not obtain lock type 2 on \\\"\\\/\\\".\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/html\\\/owncloud\\\/apps\\\/files_locking\\\/lib\\\/lockingwrapper.php(44): OCA\\\\Files_Locking\\\\Lock->addLock(2, NULL)\\n#1 \\\/var\\\/www\\\/html\\\/owncloud\\\/apps\\\/files_locking\\\/lib\\\/lockingwrapper.php(93): OCA\\\\Files_Locking\\\\LockingWrapper->getLock('thumbnails\\\/10\\\/2...', 2)\\n#2 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/files\\\/view.php(936): OCA\\\\Files_Locking\\\\LockingWrapper->file_put_contents('thumbnails\\\/10\\\/2...', '?PNG????????IHD...')\\n#3 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/files\\\/view.php(554): OC\\\\Files\\\\View->basicOperation('file_put_conten...', 'thumbnails\\\/10\\\/2...', Array, '?PNG????????IHD...')\\n#4 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/preview.php(856): OC\\\\Files\\\\View->file_put_contents('thumbnails\\\/10\\\/2...', '?PNG????????IHD...')\\n#5 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/preview.php(816): OC\\\\Preview->storeMaxPreview('thumbnails\\\/10\\\/')\\n#6 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/preview.php(521): OC\\\\Preview->generatePreview(10)\\n#7 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/preview.php(544): OC\\\\Preview->getPreview()\\n#8 \\\/var\\\/www\\\/html\\\/owncloud\\\/core\\\/ajax\\\/preview.php(61): OC\\\\Preview->showPreview()\\n#9 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/route\\\/route.php(153) : runtime-created function(1): require_once('\\\/var\\\/www\\\/html\\\/o...')\\n#10 [internal function]: __lambda_func(Array)\\n#11 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/route\\\/router.php(272): call_user_func('?lambda_230', Array)\\n#12 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/base.php(860): OC\\\\Route\\\\Router->match('\\\/core\\\/preview.p...')\\n#13 \\\/var\\\/www\\\/html\\\/owncloud\\\/index.php(40): OC::handleRequest()\\n#14 {main}\",\"File\":\"\\\/var\\\/www\\\/html\\\/owncloud\\\/apps\\\/files_locking\\\/lib\\\/lock.php\",\"Line\":251}","level":4,"time":"2015-05-05T16:48:06+00:00"}

```
### Server Host info:

Ubuntu 14.04
PHP 5.5.9
MySQL DB
### Server configuration:

ownCloud Enterprise Edition 8.1 alpha 3 (daily) Build:2015-05-05T03:08:44+00:00 fa6a67ccc1a7919aea2b18b445c0a31546184326

**Comment by jnfrmarks** [comment]
A bit more info:

The file is uploaded but there isn't a preview icon:

![p339](https://cloud.githubusercontent.com/assets/8480343/7477989/903c6d8a-f325-11e4-9041-45774ddeaeb3.PNG)

**Comment by jnfrmarks** [comment]
@DeepDiver1975 

I don't know who is looking at these issues, so ... tag you're it?

**Comment by jnfrmarks** [comment]
I can't tell if this is the root of other issues I'm seeing such as core/issues/13566

**Comment by DeepDiver1975** [comment]
@jnfrmarks should be fixed with #25 as well

**Comment by DeepDiver1975** [comment]
> as core/issues/13566

refs https://github.com/owncloud/core/issues/13566
